portfolio/pom.xml

```diff
@@ -23,7 +23,14 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
```

Comment:
Seems like this new project dependency isn't being used, but that might change as a result of other comments

Reply:
I was trying out Gson but ended up not using it for what this update did. I've removed it since it is currently not in use.